.github/workflows/pull-request-ci.yml

```diff
@@ -0,0 +1,33 @@
+
+name: CI for New Pull Requests
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main    
+
+env: 
+  DATABASE_SCHEMA: ${{ secrets.DATABASE_SCHEMA }}
```

Comment:
I realize, the DATABASE_SCHEMA and DATABASE_URL env values can't be set for the PRs because the real values are secrets and can't be set for the repository account of the creator of the PR.

If you notice https://github.com/Greenstand/treetracker-microservice-template/blob/master/.github/workflows/pull-request-ci.yml.example, the PR request installs a postgres db, runs db-migrate and runs the tests. I was wondering such a set up for the PR.